openmetadata-ui/src/main/resources/ui/src/utils/GlossaryUtils.ts

```diff
@@ -11,18 +11,46 @@
  *  limitations under the License.
  */
 
-import { FormatedGlossarySuggestion } from 'Models';
+import {
+  FormattedGlossarySuggestion,
+  FormattedGlossaryTermData,
+  SearchResponse,
+} from 'Models';
 import { DataNode } from 'rc-tree/lib/interface';
 import { GlossaryTerm } from '../generated/entity/data/glossaryTerm';
 import { ModifiedGlossaryData } from '../pages/GlossaryPage/GlossaryPageV1.component';
 import { getNameFromFQN } from './CommonUtils';
+import { searchData } from '../axiosAPIs/miscAPI';
+import { SearchIndex } from '../enums/search.enum';
+import { formatSearchGlossaryTermResponse } from './APIUtils';
+import { WILD_CARD_CHAR } from '../constants/char.constants';
+import { AxiosError } from 'axios';
 
 export interface GlossaryTermTreeNode {
   children?: GlossaryTermTreeNode[];
   fullyQualifiedName: string;
   name: string;
 }
 
+export const fetchGlossaryTerms = (): Promise<FormattedGlossaryTermData[]> => {
+  return new Promise<FormattedGlossaryTermData[]>((resolve, reject) => {
+    searchData(WILD_CARD_CHAR, 1, 1000, '', '', '', SearchIndex.GLOSSARY)
```

Comment:
Should be `''` instead of `WILD_CARD_CHAR` as this will hit api with `***` instead of just `*`